### ISSUE TITLE
Implement Stylesheet injection loader plugin

### ISSUE THREAD

**Issue by eheasley** [dojo/dom#13]
BLOCKED BY: loader implementation

This would essentially adopt xstyle's `css` plugin (and necessary logic from `xstyle/core/load-css`).

While expressing stylesheets as dependencies is convenient, it is worth noting that there are several caveats that come along with the idea of dynamically injecting stylesheets.  See the rationales section of the proposal for more information.  We'd be interested in hearing the community's opinions regarding interest for this feature given the pros and cons.

[dom Proposal Reference](https://docs.google.com/document/d/1fXGrlsl3aTqst7mCj3XqlKdbuCJs4rB9r7qIxC-tWMQ/edit#heading=h.oefy915wr12z)

**Comment by kitsonk** [comment]
Closing this in favour of dojo/meta#28
